src/python/fuzzing/fuzzer_selection.py

```diff
@@ -62,6 +63,36 @@ def update_mappings_for_fuzzer(fuzzer, mappings=None):
   ndb_utils.delete_multi([m.key for m in list(old_mappings.values())])
 
 
+def update_mappings_for_job(job, mappings):
+  """Clear existing mappings for a job, and replace them."""
+  existing_fuzzers_query = data_types.Fuzzer.query(
+      data_types.Fuzzer.jobs == job.name)
+  existing_fuzzers = {fuzzer.name: fuzzer for fuzzer in existing_fuzzers_query}
+
+  for fuzzer_name in mappings:
+    fuzzer = existing_fuzzers.pop(fuzzer_name, None)
+    if fuzzer:
+      continue
+
+    fuzzer = data_types.Fuzzer.query(
+        data_types.Fuzzer.name == fuzzer_name).get()
+    if not fuzzer:
+      logs.log_error('An unknown fuzzer %s was selected for job %s.' %
+                     (fuzzer_name, job.name))
+      continue
+
+    fuzzer.jobs.append(job.name)
+    fuzzer.put()
```

Comment:
Let's use a single put_multi that puts both these and the ones on line 93.

Reply:
done 👍